### DISCUSSION TITLE
useFlexLayout should support fixed width columns

### DISCUSSION THREAD

**Discussion by Metavirulent** [TanStack/table#2136] [General]
It is currently not possible to define a fixed width for a column using flex layout.
A common use case is that a column contains a checkbox for example and should not grow.

If minWidth and maxWidth are the same, then the corresponding flex should not be 'width 0 auto' but '0 0 width' so it doesn't grow.

**Comment by jitenderchand** [comment]
Facing the same issue

**Comment by coreybruyere** [comment]
@Metavirulent  @jitenderchand - you ever find a work around for this?

**Reply by sashaslow** [reply]
u can use ```canResize: false```
